_locales/it/messages.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": 
+  {
+    "message": "Tabs Aside",
+    "description": "name of the extension. Displayed in the manifest and pane header"
+  },
+  "description": 
+  {
+    "message": "La funzione del classico Microsoft Edge \"Tabs Aside\"",
+    "description": "Extension description"
+  },
+  "author": 
+  {
+    "message": "Michael \"XFox\" Gordeev",
+    "description": "Author name"
+  },
+  "options": 
+  {
+    "message": "Opzioni",
+    "description": "Alternative text for options button in the pane"
+  },
+  "closePanel": 
+  {
+    "message": "Chiudi",
+    "description": "Alternative text for close panel button"
+  },
+  "loadOnRestore": 
+  {
+    "message": "Riapri le schede al ripristino",
+    "description": "Label for option"
+  },
+  "showDeleteDialog": 
+  {
+    "message": "Mostra una finestra di conferma prima di eliminare qualcosa",
+    "description": "Label for option"
+  },
+  "swapIconAction": 
+  {
+    "message": "Metti da parte le schede in Tabs Aside al click sull'estensione (%TOGGLE_SHORTCUT% o click con il tasto destro sull'icona dell'estensione per aprire il pannello)",
+    "description": "Label for option"
+  },
+  "github": 
+  {
+    "message": "Visita la pagina di GitHub",
+    "description": "Link title"
+  },
+  "changelog": 
+  {
+    "message": "Changelog",
```

Comment:
Same stuff here. Just think how it will be for someone who doesn't know English. Just imagine opening a website where half of text is written in Chinese or some other language you don't know.

Actually "changelog" consists of two words: "change" and "log". So, it's something like that: `Changelog = change log = list of changes`, and you can use any of those to translate the string from

Reply:
I know, I'll translate changelog but most of the time that word isn't translated.